RELEASE/scripts/autoscend/paths/legacy_of_loathing.ash

```diff
@@ -215,4 +219,45 @@ void auto_LegacyOfLoathingDailies()
 	{
 		use(1, $item[replica Smith\'s Tome]); // get items
 	}
+
+	if(item_amount($item[Replica 2002 Mr. Store Catalog]) > 0)
+	{
+		use(1, $item[Replica 2002 Mr. Store Catalog]); //get catalog credits
+	}
+
+	pullXWhenHaveY($item[Breathitin&trade;], 1, 0);
```

Comment:
Just using this may be a bit dangerous for all paths at the beginning of a run, because a charge will be wasted if the fight is free for some other reason and unrestricted has a ton of free kills and one of the first zones we go to is outdoors.

Reply:
Moved to level_05.ash to help with finding the encryption key